.golangci.yml

```diff
@@ -1,6 +1,4 @@
 run:
-  skip-dirs:
-    - test-cmds
```

Comment:
Is this intentional? (I usually use `test-cmds` as non-committed place to test library interfaces)

Though it should also be in .gitignore